v3/plugins/broadcommfd/debugger-for-mainframe/1.2.0/meta.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+publisher: broadcommfd
+name: debugger-for-mainframe
+version: 1.2.0
+type: VS Code extension
+displayName: Debugger for Mainframe
+title: Debugger for Mainframe
+description: Provides a modern debugging experience for COBOL applications running in a CICS region.
+icon: https://open-vsx.org/api/BroadcomMFD/debugger-for-mainframe/1.2.0/file/logo.png
+repository: https://github.com/BroadcomMFD/debugger-for-mainframe
```

Comment:
I don't think we actually have an entry for this plugin in [vscode-extensions.json](https://github.com/eclipse/che-plugin-registry/blob/master/vscode-extensions.json), can you add it? The list of entries is sorted alphabetically by publisher/repo org.

Reply:
@ericwill Probably the reason why it's not listed in vscode-extensions.json is that it's not an open-source project. The repository holds just project information and releases, but not the source code, so it cannot be built from the repo. I'm not sure you want this project in the vscode-extensions.json?

Reply:
We use the `vscode-extensions.json` file to check the package.json of the extension, and do some checks. If there is no source code available upstream then adding it is pointless, yes.